daprdocs/content/en/operations/performance-and-scalability/perf-service-invocation.md

```diff
@@ -75,18 +75,10 @@ There are a number of variants that affect the CPU and memory consumption for ea
 
 ### Data plane performance
 
-The Dapr sidecar uses 0.48 vCPU and 23Mb per 1000 requests per second.
-End-to-end, the Dapr sidecars (client and server) add ~1.40 ms to the 90th percentile latency, and ~2.10 ms to the 99th percentile latency. End-to-end here is a call from one app to another app receiving a response. This is shown by steps 1-7 in [this diagram]({{< ref service-invocation-overview.md >}}).
-
-This performance is on par or better than commonly used service meshes.
-
-### Latency
-
 In the test setup, requests went through the Dapr sidecar both on the client side (serving requests from the load tester tool) and the server side (the target app).
 mTLS and telemetry (tracing with a sampling rate of 0.1) and metrics were enabled on the Dapr test, and disabled for the baseline test.
 
-<img src="/images/perf_invocation_p90.png" alt="Latency for 90th percentile">
-
-<br>
+The Dapr sidecar uses 0.45 vCPU and 38Mb per 1000 requests per second.
+End-to-end, the Dapr sidecars (client and server) add ~1.20 ms to the 90th percentile latency, and ~2.50 ms to the 99th percentile latency. End-to-end here is a call from one app to another app receiving a response. This is shown by steps 1-7 in [this diagram]({{< ref service-invocation-overview.md >}}).
 
-<img src="/images/perf_invocation_p99.png" alt="Latency for 99th percentile">
+This performance is on par or better than commonly used service meshes.
```

Comment:
Let's quantify this with a link to service meshes perf results, otherwise this is hearsay and we should not include this

Reply:
In removing the graphs, there is not mTLS measurement comparison. What impact does mTLS have on the performance tests, it would be good to at least clarify this with a % increase. The majority of people will use mTLS and so this number is important.